src/styles/utils/variebles.scss

```diff
@@ -0,0 +1,6 @@
+$color-active: #00acdc;
+$main-text-color: #060b35;
+$secondary-text-color: rgba(97, 96, 112, 1);
+$main-bg-color: rgba(255, 255, 255, 1);
+$border-color: rgba(243, 243, 243, 1);
+$color-action: #00acdc;
```

Comment:
The `$color-active` and `$color-action` variables have the same value. It's better to reuse variables for the same colors to make it easier to manage the color palette. Consider removing one of the variables and replacing its usages with the other variable.